packages/api/src/task/task.ts

```diff
@@ -84,21 +88,65 @@ export default class TaskScheduler {
             status: "ready",
           });
 
-          await db.task.update(task.id, {
-            status: {
-              phase: "completed",
-              updatedAt: Date.now(),
-            },
-            output: event.output,
-          });
           return true;
         }
       }
-      console.log(`task type unknown: ${event.task.type}`);
+
+      if (allowedTasks.includes(event.task.type)) {
```

Comment:
What is this check for? Shouldn't we update every task with its output even if we seemingly don't support it here? e.g. version migration

If we don't understand what is a given event, we should probably `nack` it instead of processing but ignoring all its information. But I think we should be fine just doing a best effort processing and at least always updating the task object.